src/gstreamer/gstkvssink.h

```diff
@@ -207,4 +212,13 @@ struct _KvsSinkMetric {
     bool on_first_frame;
 };
 
+static bool inline put_fragment_metadata(GstElement* element, const std::string name, const std::string value, bool persistent) {
```

Comment:
Curious if there is a reason this is inline.

Reply:
I needed that at one point during the development and I assumed that I needed it later. I checked that I don't need it anymore. Removed it.